packages/core/content-manager/server/src/controllers/validation/relations.ts

```diff
@@ -4,12 +4,14 @@ const validateFindAvailableSchema = yup
   .object()
   .shape({
     component: yup.string(),
-    entityId: yup.strapiID(),
+    id: yup.strapiID(),
```

Comment:
Just a reminder to switch this to document id in https://github.com/strapi/strapi/pull/19573